upf/upf.h

```diff
@@ -497,9 +497,30 @@ typedef enum
   UPF_ASSOC_COUNTER = 0,
   UPF_SESSIONS_COUNTER = 1,
   UPF_FLOW_COUNTER = 2,
-  UPF_N_COUNTERS = 3,
+  UPF_FLOW_STITCHED = 3,
+  UPF_FLOW_NOT_STITCHED_FIFOS = 4,
+  UPF_FLOW_NOT_STITCHED_NO_REV_CONNECTION = 5,
+  UPF_FLOW_NOT_STITCHED_NO_TRANSPORT = 6,
+  UPF_FLOW_NOT_STITCHED_DONT_STITCH = 7,
+  UPF_FLOW_NOT_STITCHED_INVALID_SESSION = 8,
+  UPF_FLOW_NOT_STITCHED_NO_RX_OR_TX = 9,
+  UPF_FLOW_NOT_STITCHED_MSS_MISMATCH = 10,
+  UPF_N_COUNTERS = 11,
 } upf_counters_type_t;
 
+#define foreach_upf_counter_name   \
+  _(ASSOC_COUNTER, total_assoc, upf)     \
+  _(SESSIONS_COUNTER, total_sessions, upf)      \
+  _(FLOW_COUNTER, total_flows, upf)             \
+  _(FLOW_STITCHED, total_stitched, upf)         \
```

Comment:
starting from `FLOW_STITCHED_`, maybe the prefix should be `FLOWS_STITCHED_` instead?